app/controllers/story.controller.js

```diff
@@ -200,3 +200,72 @@ exports.findAllByUser = async (req, res) => {
     }
 }
 
+
+// Add a chat to a story
+
+exports.addChat = async (req, res) => {
+
+    if (req.body.message === undefined) {
+        const error = new Error("Message cannot be empty for chat!");
+        error.statusCode = 400;
+        res.status(400).send({
+            message:
+                error.message || "Some error occurred while adding chat to story.",
+        });
+
+    }
+    const id = req.params.id;
+
+    try {
+        const chat = {
+            role: "User",
+            message: req.body.message,
+            storyId: id,
+        };
+        const history = await Chat.findAll({ where: { storyId: id } });
+
+
+        const chohereResponse = await axios.post('https://api.cohere.com/v1/chat', {
```

Comment:
`chohereResponse` should probably be spelled `cohereResponse` :)

Reply:
Modified.